src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/repository/db/EventToSendRepository.java

```diff
@@ -18,10 +18,13 @@
    *
    * @param limit the maximum number of items to return
    * @return a stream of <code>EventToSend</code> items that had previously not been claimed for
-   *     processing, but now they are claimed and locked for this transaction context
+   *     processing, but now they are claimed and locked for this transaction context, ordered by
+   *     insertion time
    */
   @Query(
-      value = "SELECT * FROM cc_schema.event_to_send LIMIT :limit FOR UPDATE SKIP LOCKED",
+      value =
+          "SELECT * FROM cc_schema.event_to_send ORDER BY cc_schema.event_to_send.insertion_time "
```

Comment:
The consequence of this is that at high volume it will incur a performance penalty.  Does this matter? Probably not, but worth being aware of considering that events are going to be idempotent anyway. 
Personally , I'm not sure it's worth bothering with this ticket.

Reply:
Was discussed and agreed to continue with ticket